src/api/app/assets/stylesheets/request_show_redesign/add_review.scss

```diff
@@ -1,5 +1,9 @@
 #review-form-collapse {
   padding: inherit;
-  left: 0;
+  right: 1.5rem;
```

Comment:
This is aligning this element with the button `Review`.

Reply:
I would go with `right: 0;`. In your examples, you are using a width of 412px. When migrating to Bootstrap, we introduced the mobile-first approach using a width of 320px. Another thing is that it makes sense nowadays.

Reply:
Okay, let's go with `right: 0;`. I find both options good.